content/handbook/product/ux/technical-writing/_index.md

```
@@ -0,0 +1,569 @@
+#### 翻訳と国際化
+
+誰でも、英語から他言語への GitLab の翻訳に貢献できます。
+GitLab の翻訳と国際化について詳しくは、[国際化](/handbook/marketing/localization/category_internationalization/) を参照してください。
+翻訳貢献のステップバイステップガイドについては、[GitLab の翻訳](https://docs.gitlab.com/development/i18n/translation/) を参照してください。
+
+[docs.gitlab.com](https://docs.gitlab.com/) サイトは英語と日本語で利用可能です。
+ローカライゼーションプロセスについて詳しくは、[プロダクトドキュメントのローカライゼーション](/handbook/marketing/localization/tech_docs_localization/) を参照してください。
+
+## アサインメント
+
+テクニカルライター（TW）は、[アサインされたグループ](#assignments-to-devops-stages-and-groups) と協力します。TW は[他の作業](#assignments-to-other-projects-and-subjects) にもアサインされる場合があります。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)